addons/modules/functions/fnc_moduleEffectFire.sqf

```diff
@@ -0,0 +1,104 @@
+#include "script_component.hpp"
+/*
+ * Author: mharis001
+ * Zeus module function to create a custom fire effect.
+ *
+ * Arguments:
+ * 0: Display <DISPLAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [DISPLAY] call zen_modules_fnc_moduleEffectFire
+ *
+ * Public: No
+ */
+
+params ["_display"];
+
+private _logic = GETMVAR(BIS_fnc_initCuratorAttributes_target,objNull);
+_display closeDisplay IDC_CANCEL; // Close helper display
+
+// Need to delay dialog creation by one frame to avoid weird input blocking bug
```

Comment:
What about other dialogs? Does it only impact this one?

Reply:
This is from the light source module which uses the same method. I think it results from closing and creating a new dialog in the same frame.